src/models/events.ts

```diff
@@ -0,0 +1,24 @@
+export enum EventAction {
+  allow = 'allow',
+  deny = 'deny',
+  challenge = 'challenge',
+};
```

Comment:
Could we actually name it just "Action" instead of the "EventAction" and keep it in the separate file. 🙄

It's actually part of the policy not the event.

Sorry for the wrong suggestion. 

Let's rename it and 🚀 it